libcaliptra/inc/caliptra_if.h

```diff
@@ -0,0 +1,40 @@
+//Licensed under the Apache-2.0 license
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define CALIPTRA_STATUS_OK 0
+
+// Memory
+
+/**
+ * caliptra_write_u32
+ *
+ * Writes a uint32_t value to the specified offset.
+ *
+ * @param[in] offset Memory offset to write
+ * @param[in] data   Data to write at offset
+ *
+ * @return 0 if successful, other if error (TBD)
+ */
+int caliptra_write_u32(uint32_t offset, uint32_t data);
+
+/**
+ * caliptra_read_u32
+ *
+ * Writes a uint32_t value to the specified offset.
```

Comment:
Comment for read says write.

Reply:
Fixed, also normalized parameter names across files.